plugins/status/metrics.go

```diff
@@ -5,6 +5,17 @@ import (
 )
 
 var (
+	createOpaInfoGauge = func(version string) prometheus.Gauge {
```

Comment:
I think we can get the current OPA version by importing `github.com/open-policy-agent/opa/version` and then using the `Version` field. If that's the case then we can do:

```go
opaInfo := prometheus.NewGauge(
	prometheus.GaugeOpts{
		Name:        "opa_info",
		Help:        "Information about the OPA environment.",
		ConstLabels: map[string]string{"version": version},
	},
    ).Set(1)
```